src/SonarQube.Client/Api/SonarQubeService.cs

```diff
@@ -233,15 +233,17 @@ public async Task<SonarQubeQualityProfile> GetQualityProfileAsync(string project
 
         public async Task<IList<SonarQubeIssue>> GetSuppressedIssuesAsync(string projectKey, CancellationToken token)
         {
+            const string statusResolved = "RESOLVED";
             var result = await InvokeRequestAsync<IGetIssuesRequest, SonarQubeIssue[]>(
                 request =>
                 {
                     request.ProjectKey = projectKey;
+                    request.Statuses = statusResolved;
                 },
                 token);
 
             return result
-                .Where(x => x.ResolutionState != SonarQubeIssueResolutionState.Unresolved)
+                .Where(x => x.Status == statusResolved)
```

Comment:
Why is this filter required? Is it because the old API doesn't support querying for specific statuses, so the results still need to be filtered? If so, it would be worth calling this out in a comment as the filter looks to be redundant.

Reply:
Also, if the filter is required for the old API, are there any unit tests that check it works as expected?